service/tpo/tpo.go

```diff
@@ -11,11 +11,11 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-var (
+const (
 	// TODO make TPO namespace configurable in separate PR.
 	DefaultNamespace = "default"
-	// TPOName is the name of the TPO the eventer watches.
-	TPOName = "draughtsman-tpo"
+	// Name is the name of the TPO the eventer watches.
+	Name = "draughtsman-tpo"
```

Comment:
Only some boyscouting. The name stuttered. Go vet should have complained.

Reply:
I think go _lint_ complains on stuttering names? i don't think vet does. Uncertain.